code/modules/modular_computers/file_system/programs/generic/ntdownloader.dm

```diff
@@ -64,26 +68,26 @@
 
 	return 1
 
-/datum/computer_file/program/ntnetdownload/proc/hide_file_info(datum/computer_file/file, skill)
-	server = (file in ntnet_global.available_station_software) ? "NTNet Software Repository" : "unspecified server"
+/datum/computer_file/program/exonetdownload/proc/hide_file_info(datum/computer_file/file, skill)
+	//server = (file in ntnet_global.available_station_software) ? "NTNet Software Repository" : "unspecified server"
```

Comment:
What's this for? 
Kinda getting tired of these random commented out bits, either reimplement or remove cleanly. Don't leave half-broken stuff in with a TODO.